app/main/views.py

```diff
@@ -10,6 +10,11 @@ def index():
     return render_template('main/index.html')
 
 
+@main.route('/map', methods=['GET', 'POST'])
+def mapusers():
```

Comment:
- let's make sure we add an `@login_required` to this method
- if we have methods with 2 words there should be an underscore in the middle, but let's change the name of this to something like `display_map` or even just `map` if you don't think that will conflict with future naming